src/Analyzers/Core/CodeFixes/AddParameter/AddParameterService.cs

```diff
@@ -130,18 +136,48 @@ public static async Task<Solution> AddParameterAsync(
                 }
 
                 if (method.MethodKind == MethodKind.ReducedExtension && insertionIndex < existingParameters.Count)
-                {
                     insertionIndex++;
+
+                var memberToAssignTo = await GetMemberToAssignToAsync(semanticDocument).ConfigureAwait(false);
```

Comment:
this ends up calling into one service to see if it can find a field/prop to assign to.